fzf-docker.plugin.zsh

```diff
@@ -17,13 +17,17 @@ _fzf_complete_docker() {
     _fzf_complete "--reverse -n 1 --height=80%" "$@" < <(
       echo $DOCKER_COMMANDS
     )
-  elif [[ $ARGS == 'docker rmi'* || $ARGS == 'docker -f'* ]]; then
-    _fzf_complete "--multi --reverse" "$@" < <(
-      docker images --format '{{.Repository}}:{{.Tag}}'
+  elif [[ $ARGS == 'docker tag'* || $ARGS == 'docker -f'* || $ARGS == 'docker run'* || $ARGS == 'docker push'* ]]; then
+    _fzf_complete "--multi --header-lines=1" "$@" < <(
+      docker images --format "table {{.Repository}}:{{.Tag}}\t{{.Size}}\t{{.ID}}\t{{.CreatedSince}}"
     )
-  elif [[ $ARGS == 'docker start'* || $ARGS == 'docker restart'* || $ARGS == 'docker stop'* || $ARGS == 'docker rm'* || $ARGS == 'docker exec'* || $ARGS == 'docker kill'* ]]; then
-    _fzf_complete "--multi --reverse" "$@" < <(
-      docker ps --format '{{.Names}}'
+  elif [[ $ARGS == 'docker rmi'* ]]; then
+    _fzf_complete "--multi --header-lines=1" "$@" < <(
+      docker images --format "table {{.ID}}\t{{.Repository}}\t{{.Tag}}\t{{.Size}}"
+    )
+  elif [[ $ARGS == 'docker stop'* || $ARGS == 'docker rm'* || $ARGS == 'docker exec'* || $ARGS == 'docker kill'* ]]; then
```

Comment:
Why did you remove `start` and `restart`?

Reply:
Ah whoops, just threw it back in.